Application/Security.js

```diff
@@ -179,7 +251,7 @@ angular.module('shared')
         currentUser = false;
 
         LoopBackAuth.clearUser();
-        LoopBackAuth.isAdmin = false;
+        //LoopBackAuth.isAdmin = false;
```

Comment:
? Why the change.  I don't have a full understanding of the LoopBackAuth module but why did this need to be taken out? Just curious.